recipes/quickfix/all/conanfile.py

```diff
@@ -11,8 +13,15 @@ class QuickfixConan(ConanFile):
     topics = ("conan", "QuickFIX", "FIX", "Financial Information Exchange", "libraries", "cpp")
     settings = "os", "compiler", "build_type", "arch"
     options = {"fPIC": [True, False],
-               "ssl":  [True, False]}
-    default_options = {"ssl": False, "fPIC": True}
+               "shared": [True, False],
```

Comment:
Is `shared` working now?

Fix the duplication of default option at line 15

Reply:
This PR actually depends on #1911 (I've been rebasing this on top of the `quickfix-static` branch). I pushed it here as a draft to see if the build was OK before I proceed.

`shared` is working in the context of #1911 , actually.

Reply:
After #1911 is merged I intend to rebase this one on the `master` branch and mark it read for review. Do you agree with this course of action?

Reply:
rebase is not required, as you only should depend of packages.